frontend/src/app/metatags.service.ts

```diff
@@ -22,11 +22,11 @@ export class MetatagsService {
     this.titleService.setTitle(title);
 
     // set default tags, that will be orverwritten below:
-    this.metaService.updateTag({ property: 'og:url', content: this.getCanonicalUrl(this.locationService.path()) });
-    this.metaService.updateTag({ property: 'og:title', content: 'Volontulo. Portal dla wolontariuszy' });
-    this.metaService.updateTag({ property: 'og:description', content: 'Volontulo. Portal dla wolontariuszy'});
-    this.metaService.updateTag({ property: 'og:image', content: this.getCanonicalUrl('/assets/img/banner/volontulo_baner.png') });
-    this.metaService.updateTag({ property: 'fb:app_id', content: environment.fbAppID });
+    this.metaService.updateTag({ name: 'og:url', content: this.getCanonicalUrl(this.locationService.path()) });
```

Comment:
These metatags should have `property` defined, not `name` (see: https://developers.facebook.com/docs/sharing/webmasters/). It was fixed in #1061. Does here merge went wrong?

Reply:
changed